README.md

```diff
@@ -19,7 +19,7 @@ If you are using Maven with [BOM][libraries-bom], add this to your pom.xml file:
     <dependency>
       <groupId>com.google.cloud</groupId>
       <artifactId>libraries-bom</artifactId>
-      <version>26.19.0</version>
+      <version>26.20.0</version>
```

Comment:
26.20.0 here, but 26.21.0 in the Groovy section below.

Reply:
This seems to be a bug in owlbot, I've filed an issue here: https://github.com/googleapis/synthtool/issues/1844